platform/ios/scripts/publish.sh

```diff
@@ -35,4 +35,15 @@ zip -r ../${ZIP} *
 step "Uploading ${ZIP} to s3…"
 REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
 aws s3 cp ../${ZIP} s3://mapbox/$REPO_NAME/ios/builds/ --acl public-read
-echo http://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}
+echo "URL: https://mapbox.s3.amazonaws.com/$REPO_NAME/ios/builds/${ZIP}"
+
+#
+# update nightly
+#
+if [[ ${PUBLISH_VERSION} =~ "nightly" ]]; then
+    step "Updating ${PUBLISH_VERSION} to ${PUBLISH_STYLE}…"
+    GENERIC_NIGHTLY_FILENAME="mapbox-ios-sdk-${PUBLISH_VERSION}.zip"
+    aws s3 cp \
+        s3://mapbox/$REPO_NAME/ios/builds/${ZIP} \
+        s3://mapbox/$REPO_NAME/ios/builds/${GENERIC_NIGHTLY_FILENAME} --acl public-read
```

Comment:
It looks like it may be possible to create a placeholder file and then the s3 API could be used here to set up a redirect to the latest version by updating the metadata of the placeholder with `x-amz-website-redirect-location`.